win32ss/user/ntuser/kbdlayout.c

```diff
@@ -6,6 +6,7 @@
  * COPYRIGHT:       Copyright 2007 Saveliy Tretiakov
  *                  Copyright 2008 Colin Finck
  *                  Copyright 2011 Rafal Harabien
+ *                  Copyright 2022 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
```

Comment:
Recall that adding copyright lines is done only for *substantial* changes to the file.